packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1414,7 +1414,12 @@ class CopyToClipboardControl extends StatelessWidget {
               ga.select(gaScreen!, gaItem!);
             }
             unawaited(
-              copyToClipboard(dataProvider!() ?? '', successMessage),
+              copyToClipboard(
+                dataProvider!() ?? '',
+                onSuccess: successMessage != null
+                    ? () => () => notificationService.push(successMessage!)
```

Comment:
Should this just be `() => notificationServer.push(successMessage!)`?

Reply:
we only want to push the message if it is non-null. `notificationService.push` expects a non-nullable value

Reply:
Oh whoops, I see the double function. Bad copy / pasting. Fixed.